controllers/update_test.go

```diff
@@ -1385,11 +1438,17 @@ func testWithCustomRepoAndImagePolicy(
 
 	// Clone the repo.
 	repoURL := gitServer.HTTPAddressWithCredentials() + repositoryPath
-	localRepo, err := clone(repoURL, "origin", args.branch)
+	localRepo, err := clone(repoURL, args.branch)
 	g.Expect(err).ToNot(HaveOccurred(), "failed to clone git repo")
+	defer localRepo.Free()
+
+	origin, err := localRepo.Remotes.Lookup("origin")
```

Comment:
Can just use `originRemote` here and some of the other code above and below as in various other place.